### PR TITLE
About Page

### PR DESCRIPTION
**Pull request by hturnbull93**
- About component added, pulling in copy from labels
- Navingation links added into NavContainer (tested)
- App restructured to allow routing between `/` and `/about` (tested)
- Fix for duplicate statue urls (when there are two statues for a single person)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
+    crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+    crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+    integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+    crossorigin="anonymous"></script>
+</body>
+
+</html>
```
Comment (benfurber) [inline public/index.html:26]: Missing line break here?

File: src/components/About.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import labels from "../labels";
+
+export function About() {
+  return (
+    <>
+      <div>{labels.aboutHeading}</div>
+      <div>{labels.aboutMissionTitle}</div>
+      <div>{labels.aboutMissionTitleText}</div>
```
Comment (benfurber) [inline src/components/About.js:9]: Would be great to spread the labels before they're returned.
E.g.:
```
export function About() {
  const { aboutHeading, aboutMissionTitle, aboutMissionTitleText } = labels
  return (
```

or:
```
import { aboutHeading, aboutMissionTitle, aboutMissionTitleText } from "../labels";
```

File: src/components/StatueMenuItem.js [hunk 3]
```diff
@@ -4,7 +4,8 @@ import style from "./StatueMenuItem.module.css";
 
 export function StatueMenuItem(props) {
   const name = props.value.name;
-  const nameNoSpaces = name.replace(/\s/g, "-");
+  const statueId = props.value.statueId;
```
Comment (benfurber) [inline src/components/StatueMenuItem.js:7]: Better to do:
```
const { name, statueId } = props.value;
```

File: src/components/StatueRoutes.spec.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```
Comment (benfurber) [inline src/components/StatueRoutes.spec.js:1]: I believe this won't be needed if the file extension is changed to jsx.

File: src/components/StatueRoutes.js [hunk 5]
```diff
@@ -6,7 +6,8 @@ import { BrowserRouter as Route, Switch } from "react-router-dom";
 export class StatueRoutes extends Component {
   render() {
     const newRoute = SlaverStatueList.map((statue, index) => {
-      const url = `/StatueDetailContainer/${statue.name.replace(/\s/g, "-")}`;
+      const slug = `${statue.name.replace(/\s/g, "-")}-${statue.statueId}`;
```
Comment (benfurber) [inline src/components/StatueRoutes.js:9]: Again, I'd prefer:
```
const { name, statueId } = statue
```

Also if it's in an object called `statue`, `statueId` should just be `id`.

File: src/components/StatueRoutes.js [hunk 6]
```diff
@@ -6,7 +6,8 @@ import { BrowserRouter as Route, Switch } from "react-router-dom";
...
+      const url = `/StatueDetailContainer/${slug}`;
```
Comment (benfurber) [inline src/components/StatueRoutes.js:10]: A line break after this please.

File: src/components/index.js [hunk 7]
```diff
@@ -3,3 +3,4 @@ export { Hero } from "./Hero.js";
 export { NavContainer } from "./NavContainer";
 export { StatueDetailContainer } from "./StatueDetailContainer";
 export { StatueRoutes } from "./StatueRoutes";
+export { About } from "./About";
```
Comment (benfurber) [inline src/components/index.js:6]: In alphabetical order please.

File: src/labels.js [hunk 8]
```diff
@@ -2,6 +2,16 @@ const labels = {
...
+  aboutMissionTitle: "Why",
+  aboutMissionTitleText:
+    "Recently a statue of Edward Colston was removed by protesters and sent into the sea. We had never known there was statue in Bristol of a slave trader. This made us realise that we walk past all this public art and have no idea about the people they represent and if these celebrated people represent values that we share. We want people to have a deeper understanding of the art that surrounds them, and have an opportunity to reflect on the values represented by the statue , the artist and if these values have a place in our cultural identity.",
+  aboutDataTitle: "Data Collection",
+  aboutData: "Pyhton blah blah",
+  aboutFrontEndTitle: "Front End",
+  aboutFrontEnd:
+    "We choose to use React partly because of the learning experience, and because as we were not sure of how large the project would be we needed something scalable.",
+  aboutThanks:
+    "Thanks to Ben Furber for his continuous support and thorough peer reviews.",
```
Comment (benfurber) [inline src/labels.js:15]: Might be better to nest all of these in an `about` parent. e.g.
```
   about: {
    header:  "About the project",
    ...
```

File: src/statues/slavers.js [hunk 9]
```diff
@@ -1,19 +1,22 @@
 const slaversStatueList = [
   {
+    statueId: 1,
```
Comment (benfurber) [inline src/statues/slavers.js:3]: Can just be `id`.

File: src/statues/slavers.js [hunk 10]
```diff
@@ -1,19 +1,22 @@
...
     name: "William Penn",
     wikiArticleURI: "https://en.wikipedia.org/wiki/William_Penn",
     artworkTitle: "William Penn",
     artworkURI: "https://artuk.org/discover/artworks/william-penn-244578/",
     artworkArtist: "Eli Johnson (1850â1881)",
   },
   {
+    statueId: 2,
```
Comment (benfurber) [inline src/statues/slavers.js:11]: Can just be `id`.

File: src/statues/slavers.js [hunk 11]
```diff
@@ -1,19 +1,22 @@
...
     name: "William Penn",
     wikiArticleURI: "https://en.wikipedia.org/wiki/William_Penn",
     artworkTitle: "William Penn",
     artworkURI: "https://artuk.org/discover/artworks/william-penn-244564/",
     artworkArtist: "Eli Johnson (1850â1881)",
   },
   {
+    statueId: 1,
```
Comment (benfurber) [inline src/statues/slavers.js:19]: etc.

File: src/components/StatueRoutes.spec.js (hunk 4 above)
Comment (hturnbull93) [reply src/components/StatueRoutes.spec.js:1]: Tried this and no dice unfortunately, but have removed the rule ignore.

File: src/statues/slavers.js (hunk 9 above)
Comment (hturnbull93) [reply src/statues/slavers.js:3]: These aren't actually unique as they only help differentiate between different artworks of the same person, but we couldn't think of a better name for them.

**Pull request merged**